### ISSUE TITLE
Some localisation strings are missing in CrowdIn

### ISSUE THREAD

**Issue by florian-h05** [openhab/openhab-addons#14486]
After adding new strings to the JS Scripting addon a while ago …

https://github.com/openhab/openhab-addons/blob/7e047b3fc672bcc140d51b999cff0b243fa2f0ba/bundles/org.openhab.automation.jsscripting/src/main/resources/OH-INF/i18n/jsscripting.properties#L5-L8

…, I noticed that these strings are still missing in CrowdIn, e.g. https://crowdin.com/translate/openhab-addons/5585/en-de?filter=basic&value=0.

@cweitkamp Do you have any idea what‘s going on here?

**Comment by florian-h05** [comment]
@kaikreuzer Sorry for pinging, but do you have an idea?

**Comment by kaikreuzer** [comment]
@florian-h05 No, I do not really have an idea. I just checked and manually sync'ed the repo, but it stayed at 5 texts in that file. I'd assume that there is some bug in the crowdin sync logic...
What I have now done is to manually upload the latest version of the file - this was accepted and all texts now appear.
While this does not solve the general issues that for certain files some strings might be missing, it at least has solved it for this file in question here.

**Comment by florian-h05** [comment]
Thanks!

I will close this issue for now, and report if I encounter more missing localisation strings.
